index.html

```diff
@@ -259,22 +260,20 @@ <h1 class="page-header">Dashboard</h1>
         <div class="col-sm-12">
           <div class="panel panel-default">
             <div class="panel-heading">
-              <i class="fa fa-bar-chart-o fa-fw"></i>
               <h4>Air Dustiness Chart</h4>
             </div>
             <div class="panel-body">
-              <div class="center-chart" id="dustinessOfTheAirChart">
+              <div style="width: 100%" class="center-chart" id="dustinessOfTheAirChart">
               </div>
             </div>
           </div>
         </div>
       </div>
 
       <div class="row">
-        <div class="col-sm-4 col-sm-offset-1">
+        <div class="col-sm-5 col-sm-offset-1">
```

Comment:
I believe the PM/boss said all gauge charts should be col-sm-6 without offsets, could you please ask him just for us to be absolutely sure?